misago/markup/tests/test_parser.py

```diff
@@ -192,7 +192,7 @@ def test_clean_current_link(self):
 """.strip()
 
         expected_result = """
-<p>Lorem ipsum: <a href="/" rel="nofollow">test.com</a></p>
```

Comment:
Hey, this change is unacceptable. Forum has to add rel="nofollow" to user posted links to protect its rating in search engines and discourage automated postings.